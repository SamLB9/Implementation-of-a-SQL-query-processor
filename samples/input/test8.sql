SELECT Car.Model, SUM(Car.Price)
FROM Car
GROUP BY Car.Model
ORDER BY SUM(Car.Price);