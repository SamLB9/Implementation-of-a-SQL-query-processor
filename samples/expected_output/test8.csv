2, 40000
3, 45000
1, 98000